multiselect.go

```diff
@@ -28,9 +29,10 @@ type MultiSelectTemplateData struct {
 	MultiSelect
 	Answer        string
 	ShowAnswer    bool
-	Checked       map[int]bool
+	Checked       map[string]bool
 	SelectedIndex int
 	ShowHelp      bool
+	Choices       []string
```

Comment:
Wondering if we should change the name of this, maybe `Page`, `PageOptions` or something like that?  `Choices` and `Options` are indistinguishable to me.